photon-api/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -131,6 +131,11 @@ class GameEstimator(val sc: SparkContext, implicit val logger: Logger) extends P
     "use warm start",
     "Whether to train the current model with coefficients initialized by the previous model.")
 
+  val interceptIndices: Param[Seq[Int]] = ParamUtils.createParam[Seq[Int]](
+    "intercept indices",
+    "A list of integers that indicate the index of the intercept in each feature bag."
+  )
```

Comment:
There's too much implicit information here - the relationship between intercept indices and coordinates should be made explicit. The indices should also be optional. We can create a default value which will return `None` for all coordinates, for the case where no input is given.
```suggestion
  val coordinateInterceptIndices: Param[Map[CoordinateId, Option[Int]]] = ParamUtils.createParam[Map[CoordinateId, Option[Int]]](
  "coordinate intercept indices",
  "A map of coordinate ID to intercept index, if enabled.")
```
However, now I wonder if we can insert this information into another class so that this parameter doesn't need to exist.